packages/bar/src/compute/totals.ts

```diff
@@ -56,7 +57,7 @@ export const computeBarTotals = <RawDatum extends BarDatum>(
                 key: 'total_' + indexValue,
                 x: xPosition,
                 y: yPosition,
-                value: indexTotal,
+                value: formatValue(indexTotal),
```

Comment:
I think we should keep both the value and the `formattedValue`, that's usually what we do in other components.